MCprep_addon/conf.py

```diff
@@ -124,6 +126,13 @@ def __init__(self):
 		# that no reading has occurred. If lib not found, will update to [].
 		# If ever changing the resource pack, should also reset to None.
 		self.material_sync_cache = []
+		
+		# i18n using Python's gettext module
+		self.en_i18 = gettext.translation("mcprep", self.languages_folder, fallback=True, languages=['en'])
```

Comment:
Is this going to feed in with Blender's native locale setting? Or do users have to somehow configure that separately. It should be a clear requirement that MCprep's translation follows suite of blender itself.

Reply:
The goal is to, by default, use the user's set locale for Blender, but we might also have our own menu if we receive translations for languages not in Blender's locale settings.